hare/broker.go

```diff
@@ -34,36 +36,44 @@ func (closer *Closer) CloseChannel() chan struct{} {
 	return closer.channel
 }
 
+const (
+	invalid = false
```

Comment:
enum-like fields should be with ALL-CAPS

Reply:
It is not GO_IDIOMATIC. Also, it will make them accessible outside the hare package - no reason to do that.